controllers/jenkins/pipelinerun/pipelinerun_controller.go

```diff
@@ -127,7 +127,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	// first run
-	pipelineBuild, err := r.triggerJenkinsJob(devopsProjectName, pipelineName)
+	pipelineBuild, err := r.triggerJenkinsJob(devopsProjectName, pipelineName, &pr.Spec)
 	if err != nil {
```

Comment:
Please consider adding a condition for this case.

Reply:
You mean, if an error is returned, add a condition into status.conditions, is that right?

Reply:
Yes

Reply:
I think we can skip this error handling since the handling won't block this PR. And I'll fire another PR to handle it. What do you reckon?